package.json

```diff
@@ -23,15 +23,19 @@
     "@nuxt/typescript-build": "2.1.0",
     "@nuxtjs/eslint-config-typescript": "6.0.1",
     "@nuxtjs/eslint-module": "3.0.2",
+    "@nuxtjs/tailwindcss": "^4.2.1",
```

Comment:
Can you use non caret dependencies? That way we can guarantee which versions of the libs are downloaded

Reply:
I don't think we should go out of our ways to change what package managers download by default unless we have security reasons to.

Reply:
Can we leave these as-is for now? Once we reach MVP we may want to lock these for stability reasons. Currently any minor updates may be beneficial.